networkx/utils/mapped_queue.py

```diff
@@ -93,27 +105,49 @@ class MappedQueue:
     library. While MappedQueue is designed for maximum compatibility with
     heapq, it adds element removal, lookup, and priority update.
 
+    Unsortable elements should be mapped to a priority via a dict.
+    Otherwise a TypeError will occur when the comparisons are made.
+
```

Comment:
Just my opinion, but I personally found this statement hard to understand until I got down to the examples section.

I'd vote to remove the note as I think it becomes much more clear with the attached example, but feel free to ignore!

Reply:
Agree. I removed the note and let the example explain why sorting breaks if node names are mixed integers and strings.